docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2906,22 +2911,26 @@ noise added to the data. We currently divide noise into two types:
 #. **Column based noise:** errors in individual data entry, such as miswriting or incorrectly selecting responses 
 #. **Row based noise:** errors in the inclusion or exclusion of entire rows of data, such as duplication or omission 
 
-As there are multiple noise functions that will independently select data, 
-the order to apply noise is: 
+As there are multiple noise functions that will independently select 
+data. Noise should be added in the order below. Note that not all 
+noise functions apply to add data types, but this ordering should 
+be able to be used for all data. 
 
+#. "Borrowed" SSN (happens in simulation NOT noise functions)
+#. Missing Data 
 #. Omissions 
 #. Duplications 
-#. Column noise is divided in a few parts, for strings: 
-    #. Nicknames 
-    #. Fake names
-    #. Phonetic 
-    #. OCR 
-    #. Typographic 
-#. Column noise for numeric data: 
-    #. Copy from within the household 
-    #. Month and day swaps (applies to dates only)
-    #. Numeric miswriting 
-#. Selection type data only has one column noise type 
+#. Incorrect Selection 
+#. Copy from Within Household 
+#. Month and day swaps (applies to dates only)
+#. Zip Code Miswriting (applies to Zip Code only)
+#. Age Miswriting (applies to age only)
+#. Nicknames 
+#. Fake names
+#. Phonetic 
+#. OCR 
+#. Typographic 
+#. Numeric miswriting 
```

Comment:
It seems like we would want numeric miswriting to happen before OCR or typographic noise, e.g. around the same stage that the zipcode miswriting or age miswriting happens. @SylLutze, is there a reason you put it last?

Reply:
I just assumed it didn't matter the order if it was also a type of noise applied to strings. I can move it to where age and zip are. Those are higher since I thought they required a different non-string datatype and therefore should go before OCR/typographic. I can just numeric there are well.

Reply:
Maybe it's not a big deal, but just thinking about the data generation process in terms of how the data might actually get corrupted, it seems to me like OCR and typographic should always come last.

Reply:
Sure, makes sense. I swapped the order.